java/src/main/native/src/map_lookup.cu

```diff
@@ -127,6 +127,35 @@ get_gather_map_for_map_values(column_view const &input, string_scalar &lookup_ke
 } // namespace
 
 namespace jni {
+
+std::unique_ptr<column> map_contains(column_view const &map_column, string_scalar lookup_key,
```

Comment:
I think it's better to reuse this function in `map_lookup` instead of duplicating codes.